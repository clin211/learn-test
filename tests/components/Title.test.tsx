import React from 'react';
import { render } from '@testing-library/react';
import Title from '../../src/components/Title';

describe('Title Component', () => {
    it('渲染文字大字', () => {
        const { getByText } = render(<Title type="large" title="大字" />);
        const content = getByText('大字');
        expect(content).toMatchSnapshot();
    });

    it('渲染文字小字', () => {
        const { getByText } = render(<Title type="small" title="小字" />);
        const content = getByText('小字');
        expect(content).toMatchSnapshot();
    });
});
