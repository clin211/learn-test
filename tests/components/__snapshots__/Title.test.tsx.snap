// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Component 渲染文字大字 1`] = `
<div>
  大字
</div>
`;

exports[`Title Component 渲染文字小字 1`] = `
<div>
  小字
</div>
`;
